src/api/v2/paths/directory.js

```diff
@@ -12,7 +12,13 @@ import { getDirectories } from '../db/ldap/directory-dao';
  */
 const get = async (req, res) => {
   try {
-    const valuelessParams = _.pickBy(req.query, (value) => !value && value !== 0);
+    const params = req.query;
+    const paramKeys = Object.keys(params);
+    if (paramKeys.includes('primaryAffiliation') && paramKeys.length === 3) {
+      return errorBuilder(res, 400, ['primaryAffiliation may not be used as a lone query parameter.']);
```

Comment:
Do we want to directly return a 400 here? For example, if we got this request `?primaryAffiliation=`, shouldn't we return both "Query parameter '${key}' must have a value." error and "primaryAffiliation may not be used as a lone query parameter." in the 400 errors list?

Reply:
I added some logic that would do this, but like we discussed last Friday, this particular case will still return just the openapi enum error unless we make some changes to the skeleton.